src/components/components/ebay-dialog-base/component.js

```diff
@@ -103,11 +104,11 @@ module.exports = {
 
                 if (isTrapped) {
                     focusEl.focus();
-                    this.emit('dialog-show');
+                    this.emit('modal-show');
```

Comment:
So these event names are internal ones? Nitpicky, but in future we might have modal and non-modal dialogs. So the name may need to be changed. I guess if it is internal only, then there's no hurry do do that.

Reply:
These are internal events only.